packages/babel-cli/bin/babel/file.js

```diff
@@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
     results = [];
 
     _.each(filenames, function (filename) {
-      if (!fs.existsSync(filename)) return;
+      if (!pathExists.sync(filename)) return;
```

Comment:
Ah yeah. I’ll do it, pushing a new release now. Thanks!

On Thu, Jun 25, 2015 at 11:09 PM, Mark Simulacrum
<notifications@github.com> wrote:

>> @@ -100,7 +100,7 @@ module.exports = function (commander, filenames, opts) {
>>      results = [];
>>  
>>      _.each(filenames, function (filename) {
>> -      if (!fs.existsSync(filename)) return;
>> +      if (!pathExists.sync(filename)) return;
> Fixed, but do I need to add path-exists to the babel-cli package.json (if it exists?).
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/babel/babel/pull/1832/files#r33310602